backend/main.py

```diff
@@ -1,14 +1,54 @@
-from fastapi import FastAPI
-from pydantic import BaseModel
+import json
+from http import HTTPStatus
 
 
-class TextModel(BaseModel):
-    text: str
+KNOWN_OPERATIONS = ['reverse']
 
 
-app = FastAPI()
+def make_return(status_code, body_text):
+    return {
+            "statusCode": status_code,
+            "body": body_text,
+            "headers": {"content-type": "application/json"},
+        }
 
 
-@app.post("/reverse/")
-async def reverse_text(item: TextModel):
-    return {"reversed": item.text[::-1]}
+def validate_request(request_body):
+    if 'operation' not in request_body:
```

Comment:
That super cool to have input request validation!
Please take a look to `Pydantic`, library, designed to do similar thing. Totally fine to add it in one of the following PRs (not now). Example how to use it: https://chat.openai.com/c/3e90a0b4-f9d0-4109-9917-19609b1c3e65